schemas/destinations/destination.schema.json

```diff
@@ -83,6 +83,16 @@
           "description": "Unique identifier to connect and deliver segments to the destination",
           "meta:titleId": "destination##xdm:connectionSpecID##title##40981",
           "meta:descriptionId": "destination##xdm:connectionSpecID##description##44091"
+        },
+        "xdm:destinationCategory": {
```

Comment:
It will help to have enums defined for this field is you have a list

Reply:
New categories get added as new destinations are worked on.

Reply:
Sounds Good

Reply:
Will update with the enum values for this field as well as the destinationFrequency.

Reply:
Pushed the enum values.